dojo/tools/neuvector/parser.py

```diff
@@ -22,82 +22,114 @@ def parse_json(self, json_output):
         try:
             data = json_output.read()
             try:
-                tree = json.loads(str(data, 'utf-8'))
-            except:
+                tree = json.loads(str(data, "utf-8"))
+            except BaseException:
```

Comment:
Change to `Exception`

Reply:
Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8305/commits/5cc690eda741b1f58c6e5d39075007108035d7a7